mteb/models/model_implementations/slm_models.py

```diff
@@ -0,0 +1,454 @@
+
+        self.processor = ColMinistral3Processor.from_pretrained(model_name)
+        
+        logger.info(f"SLM-ColMinistral3 loaded: dim={self.mdl.dim}, device={self.device}")
+
+
+# =============================================================================
+# Citations
+# =============================================================================
+
+SAUERKRAUTLM_CITATION = """
+@misc{sauerkrautlm-colpali-2025,
+  title={SauerkrautLM-ColPali: Multi-Vector Vision Retrieval Models},
+  author={David Golchinfar},
+  organization={VAGO Solutions},
+  year={2025},
+  url={https://github.com/VAGOsolutions/sauerkrautlm-colpali}
+}
+"""
+
+COLPALI_CITATION = """
+@misc{faysse2024colpali,
+  title={ColPali: Efficient Document Retrieval with Vision Language Models},
+  author={Faysse, Manuel and Sibille, Hugues and Wu, Tony and Omrani, Bilel and Viaud, Gautier and Hudelot, C\\'eline and Colombo, Pierre},
+  year={2024},
+  eprint={2407.01449},
+  archivePrefix={arXiv},
+  primaryClass={cs.IR}
+}
+"""
+
+
+# =============================================================================
+# ColQwen3 Model Metadata
+# =============================================================================
+
+# ColQwen3-1.7B Turbo: ~1.7B params → 3.4 GB VRAM in bfloat16
+slm_colqwen3_1_7b_turbo = ModelMeta(
+    loader=SLMColQwen3Wrapper,
+    name="VAGOsolutions/SauerkrautLM-ColQwen3-1.7b-Turbo-v0.1",
+    languages=SUPPORTED_LANGUAGES,
+    revision="19c295a18e057d6d82754f627c09408117ffdb66",
+    release_date="2025-12-20",
+    modalities=["image", "text"],
+    n_parameters=1_700_000_000,
+    memory_usage_mb=3400,
+    max_tokens=262144,
+    embed_dim=128,
+    license="apache-2.0",
+    open_weights=True,
+    public_training_code=None,
+    public_training_data=None,
+    framework=["ColPali"],
+    reference="https://huggingface.co/VAGOsolutions/SauerkrautLM-ColQwen3-1.7b-Turbo-v0.1",
+    similarity_fn_name=ScoringFunction.MAX_SIM,
+    use_instructions=True,
+    adapted_from="Qwen/Qwen3-VL-2B-Instruct",
+    training_datasets={"vidore/colpali_train_set"},
+    citation=SAUERKRAUTLM_CITATION + COLPALI_CITATION,
+)
+
+# ColQwen3-2B: ~2.2B params → 4.4 GB VRAM in bfloat16
+slm_colqwen3_2b = ModelMeta(
+    loader=SLMColQwen3Wrapper,
+    name="VAGOsolutions/SauerkrautLM-ColQwen3-2b-v0.1",
+    languages=SUPPORTED_LANGUAGES,
+    revision="48f699713c10af754684e12060a2af9266462cc9",
+    release_date="2025-12-20",
+    modalities=["image", "text"],
+    n_parameters=2_200_000_000,
+    memory_usage_mb=4400,
+    max_tokens=262144,
+    embed_dim=128,
+    license="apache-2.0",
+    open_weights=True,
+    public_training_code=None,
+    public_training_data=None,
+    framework=["ColPali"],
+    reference="https://huggingface.co/VAGOsolutions/SauerkrautLM-ColQwen3-2b-v0.1",
+    similarity_fn_name=ScoringFunction.MAX_SIM,
+    use_instructions=True,
+    adapted_from="Qwen/Qwen3-VL-2B-Instruct",
+    training_datasets={"vidore/colpali_train_set"},
```

Comment:
There should be mostly MTEB tasks. Please, change to
```suggestion
    training_datasets={"MMarcoReranking", "VDRMultilingualRetrieval"} | COLPALI_TRAINING_DATA,
```